js/popups/tablemenupopup.ts

```diff
@@ -334,6 +340,12 @@ export class TableMenuPopup {
             leaved.then(removeCurrentTable);
         }
     }
+    private toggleSkipDeals() {
+        this.skipDeals(!this.skipDeals());
+        const currentIndex = tableManager.currentIndex();
+        const tableView = tableManager.tables()[currentIndex];
```

Comment:
Please use `const currentTable = this.currentTableProvider.currentTable();` for getting reference to current table